ueransim/entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ex
+
+if [ $# -lt 1 ]
+then
+        echo "Usage : $0 [gnb|ue]"
+        exit
+fi
+
+export AMF_ADDR="$(host -4 amf |awk '/has.*address/{print $NF; exit}')"
```

Comment:
the "amf" and "gnb" hostnames must be configurable through env variables.
Take a look to how srs-lte does with mme and enb: 
```
if [[ ! -z "$ENB_HOSTNAME" ]] ; then 
    export ENB_ADDRESS="$(host -4 $ENB_HOSTNAME |awk '/has.*address/{print $NF; exit}')"
fi
```